### ISSUE TITLE
Ench : Another contact person for customer / people ?

### ISSUE THREAD

**Issue by likozhang** [jekkos/opensourcepos#674]
more customer not use only phone number and email
may another like (BBM,Line,Whatsapp, etc)

how about, add " other contact "

**Comment by daN4cat** [comment]
Yes, feel free to add and suggest. Whatsapp works with your phone number so you don't really need more than that. Other means could be a nick name on some other social media channel.
